spec/requests/api/v1/sessions/create_spec.rb

```diff
@@ -25,20 +27,38 @@
       expect(response).to be_successful
     end
 
-    it 'returns the user' do
+    it 'returns the user id' do
       expect(json[:user][:id]).to eq(user.id)
+    end
+
+    it 'returns the user email' do
       expect(json[:user][:email]).to eq(user.email)
+    end
+
+    it 'returns the user username' do
       expect(json[:user][:username]).to eq(user.username)
+    end
+
+    it 'returns the user uid' do
       expect(json[:user][:uid]).to eq(user.uid)
+    end
+
+    it 'returns the user provider' do
       expect(json[:user][:provider]).to eq('email')
+    end
+
+    it 'returns the user first name' do
       expect(json[:user][:first_name]).to eq(user.first_name)
+    end
+
+    it 'returns the user last name' do
       expect(json[:user][:last_name]).to eq(user.last_name)
     end
```

Comment:
maybe would be better to keep it as it was and add `:aggregate_failures` ?

https://docs.rubocop.org/rubocop-rspec/cops_rspec.html#aggregate_failures-true-default